addons/binding/org.openhab.binding.airvisualnode/src/main/java/org/openhab/binding/airvisualnode/AirVisualNodeBindingConstants.java

```diff
@@ -8,44 +8,46 @@
  */
 package org.openhab.binding.airvisualnode;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.Set;
 
-import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.smarthome.core.thing.ThingTypeUID;
 
-import com.google.common.collect.ImmutableSet;
-
 /**
  * The {@link AirVisualNodeBindingConstants} class defines common constants, which are
  * used across the whole binding.
  *
  * @author Victor Antonovich - Initial contribution
  */
+@NonNullByDefault
 public class AirVisualNodeBindingConstants {
 
-    @NonNull public static final String BINDING_ID = "airvisualnode";
+    public static final String BINDING_ID = "airvisualnode";
 
     // List of all Thing Type UIDs
     public static final ThingTypeUID THING_TYPE_AVNODE = new ThingTypeUID(BINDING_ID, "avnode");
 
     // List of all Channel ids
-    public static final String CHANNEL_CO2_PPM = "co2_ppm";
+    public static final String CHANNEL_CO2 = "co2";
     public static final String CHANNEL_HUMIDITY = "humidity";
-    public static final String CHANNEL_AQI_CN = "aqi_cn";
-    public static final String CHANNEL_AQI_US = "aqi_us";
+    public static final String CHANNEL_AQI_US = "aqi";
     public static final String CHANNEL_PM_25 = "pm_25";
-    public static final String CHANNEL_TEMP_CELSIUS = "temp_celsius";
-    public static final String CHANNEL_TEMP_FAHRENHEIT = "temp_fahrenheit";
+    public static final String CHANNEL_TEMP_CELSIUS = "temperature";
     public static final String CHANNEL_BATTERY_LEVEL = "battery_level";
-    public static final String CHANNEL_WIFI_STRENGTH = "wifi_strength";
+    public static final String CHANNEL_WIFI_STRENGTH = "signal_strength";
```

Comment:
Since you now use the system channels, its probably better to use their definitions directly, i.e.: `DefaultSystemChannelTypeProvider.SYSTEM_SYSTEM_CHANNEL_SIGNAL_STRENGTH.getUID().getId()`

Same applies to other system channels of course.